api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -110,8 +110,13 @@ public void checkFreeTierBillingUsage() {
       deactivateUserWorkspaces(user);
     }
 
-    sendAlertsForCostThresholds(previousUserCosts, userCosts, currentExpiredUsers);
-    sendAlertsForTimeThresholds(userCosts, currentExpiredUsers);
+    final Set<DbUser> usersWithNonNullRegistration =
+        userDao.findByFirstRegistrationCompletionTimeNotNull();
+    final Set<DbUser> usersToThresholdCheck =
+        Sets.difference(usersWithNonNullRegistration, currentExpiredUsers);
+
+    sendAlertsForCostThresholds(usersToThresholdCheck, previousUserCosts, userCosts);
```

Comment:
This is a change: previously we would run this cost check for users with null registration times.

However it is not very meaningful because any user who actually logs into the system will not have a null registration time.

I decided to make this change because it makes the logic a little simpler.